models/UserSearch.php

```diff
@@ -56,7 +59,8 @@ public function __construct(Finder $finder, $config = [])
     public function rules()
     {
         return [
-            'fieldsSafe' => [['id', 'username', 'email', 'registration_ip', 'created_at', 'last_login_at'], 'safe'],
+
```

Comment:
I think it is for easier override of the application-specific User Model. I use it.